File: client/iih_stub/iih.ci_stub/iih.ci.ord/dto/ordprintparamdto/d/OrdPrintParamDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.ord.dto.ordprintparamdto.d
{
    /// <summary>
    /// OrdPrintParamDTO 的摘要说明。
    /// </summary>
    public class OrdPrintParamDTO : BaseDTO {

        public OrdPrintParamDTO() {
 
        }

        /// <summary>
        /// 长临标识
        /// </summary>
		public bool? Fg_long {
            get { return getAttrVal<FBoolean>("Fg_long",null); }
            set { setAttrVal<FBoolean>("Fg_long", value); }
        }

        /// <summary>
        /// 就诊ID
        /// </summary>
		public string Id_en {
            get { return getAttrVal<string>("Id_en",null); }
            set { setAttrVal<string>("Id_en", value); }
        }

        /// <summary>
        /// 就诊号
        /// </summary>
		public string Code_en {
            get { return getAttrVal<string>("Code_en",null); }
            set { setAttrVal<string>("Code_en", value); }
        }

        /// <summary>
        /// 患者姓名
        /// </summary>
		public string Name_en {
            get { return getAttrVal<string>("Name_en",null); }
            set { setAttrVal<string>("Name_en", value); }
        }

        /// <summary>
        /// 患者ID
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 打印模式
        /// </summary>
		public string Print_mode {
            get { return getAttrVal<string>("Print_mode",null); }
            set { setAttrVal<string>("Print_mode", value); }
        }

        /// <summary>
        /// 页号
        /// </summary>
		public int? Page_num {
            get { return getAttrVal<int?>("Page_num",null); }
            set { setAttrVal<int?>("Page_num", value); }
        }

        /// <summary>
        /// 就诊类型ID
        /// </summary>
		public string Id_entp {
            get { return getAttrVal<string>("Id_entp",null); }
            set { setAttrVal<string>("Id_entp", value); }
        }

        /// <summary>
        /// 就诊类型编码
        /// </summary>
		public string Code_entp {
            get { return getAttrVal<string>("Code_entp",null); }
            set { setAttrVal<string>("Code_entp", value); }
        }

        /// <summary>
        /// 就诊科室ID
        /// </summary>
		public string Id_dep_phy {
            get { return getAttrVal<string>("Id_dep_phy",null); }
            set { setAttrVal<string>("Id_dep_phy", value); }
        }

        /// <summary>
        /// 就诊科室
        /// </summary>
		public string Name_dep_phy {
            get { return getAttrVal<string>("Name_dep_phy",null); }
            set { setAttrVal<string>("Name_dep_phy", value); }
        }

        /// <summary>
        /// 就诊病区ID
        /// </summary>
		public string Id_dep_nur {
            get { return getAttrVal<string>("Id_dep_nur",null); }
            set { setAttrVal<string>("Id_dep_nur", value); }
        }

        /// <summary>
        /// 就诊病区
        /// </summary>
		public string Name_dep_nur {
            get { return getAttrVal<string>("Name_dep_nur",null); }
            set { setAttrVal<string>("Name_dep_nur", value); }
        }

        /// <summary>
        /// 住院号
        /// </summary>
		public string Code_amr_ip {
            get { return getAttrVal<string>("Code_amr_ip",null); }
            set { setAttrVal<string>("Code_amr_ip", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Fg_long", "Id_en", "Code_en", "Name_en", "Id_pat", "Print_mode", "Page_num", "Id_entp", "Code_entp", "Id_dep_phy", "Name_dep_phy", "Id_dep_nur", "Name_dep_nur", "Code_amr_ip"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.ord.dto.ordprintparamdto.d.OrdPrintParamDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/ord/dto/ordprintparamdto/d/OrdPrintParamDTO.java
package iih.ci.ord.dto.ordprintparamdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 医嘱单打印参数模型dto DTO数据 
 * 
 */
public class OrdPrintParamDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 长临标识
	 * @return FBoolean
	 */
	public FBoolean getFg_long() {
		return ((FBoolean) getAttrVal("Fg_long"));
	}
	/**
	 * 长临标识
	 * @param Fg_long
	 */
	public void setFg_long(FBoolean Fg_long) {
		setAttrVal("Fg_long", Fg_long);
	}
	/**
	 * 就诊ID
	 * @return String
	 */
	public String getId_en() {
		return ((String) getAttrVal("Id_en"));
	}
	/**
	 * 就诊ID
	 * @param Id_en
	 */
	public void setId_en(String Id_en) {
		setAttrVal("Id_en", Id_en);
	}
	/**
	 * 就诊号
	 * @return String
	 */
	public String getCode_en() {
		return ((String) getAttrVal("Code_en"));
	}
	/**
	 * 就诊号
	 * @param Code_en
	 */
	public void setCode_en(String Code_en) {
		setAttrVal("Code_en", Code_en);
	}
	/**
	 * 患者姓名
	 * @return String
	 */
	public String getName_en() {
		return ((String) getAttrVal("Name_en"));
	}
	/**
	 * 患者姓名
	 * @param Name_en
	 */
	public void setName_en(String Name_en) {
		setAttrVal("Name_en", Name_en);
	}
	/**
	 * 患者ID
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者ID
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 打印模式
	 * @return String
	 */
	public String getPrint_mode() {
		return ((String) getAttrVal("Print_mode"));
	}
	/**
	 * 打印模式
	 * @param Print_mode
	 */
	public void setPrint_mode(String Print_mode) {
		setAttrVal("Print_mode", Print_mode);
	}
	/**
	 * 页号
	 * @return Integer
	 */
	public Integer getPage_num() {
		return ((Integer) getAttrVal("Page_num"));
	}
	/**
	 * 页号
	 * @param Page_num
	 */
	public void setPage_num(Integer Page_num) {
		setAttrVal("Page_num", Page_num);
	}
	/**
	 * 就诊类型ID
	 * @return String
	 */
	public String getId_entp() {
		return ((String) getAttrVal("Id_entp"));
	}
	/**
	 * 就诊类型ID
	 * @param Id_entp
	 */
	public void setId_entp(String Id_entp) {
		setAttrVal("Id_entp", Id_entp);
	}
	/**
	 * 就诊类型编码
	 * @return String
	 */
	public String getCode_entp() {
		return ((String) getAttrVal("Code_entp"));
	}
	/**
	 * 就诊类型编码
	 * @param Code_entp
	 */
	public void setCode_entp(String Code_entp) {
		setAttrVal("Code_entp", Code_entp);
	}
	/**
	 * 就诊科室ID
	 * @return String
	 */
	public String getId_dep_phy() {
		return ((String) getAttrVal("Id_dep_phy"));
	}
	/**
	 * 就诊科室ID
	 * @param Id_dep_phy
	 */
	public void setId_dep_phy(String Id_dep_phy) {
		setAttrVal("Id_dep_phy", Id_dep_phy);
	}
	/**
	 * 就诊科室
	 * @return String
	 */
	public String getName_dep_phy() {
		return ((String) getAttrVal("Name_dep_phy"));
	}
	/**
	 * 就诊科室
	 * @param Name_dep_phy
	 */
	public void setName_dep_phy(String Name_dep_phy) {
		setAttrVal("Name_dep_phy", Name_dep_phy);
	}
	/**
	 * 就诊病区ID
	 * @return String
	 */
	public String getId_dep_nur() {
		return ((String) getAttrVal("Id_dep_nur"));
	}
	/**
	 * 就诊病区ID
	 * @param Id_dep_nur
	 */
	public void setId_dep_nur(String Id_dep_nur) {
		setAttrVal("Id_dep_nur", Id_dep_nur);
	}
	/**
	 * 就诊病区
	 * @return String
	 */
	public String getName_dep_nur() {
		return ((String) getAttrVal("Name_dep_nur"));
	}
	/**
	 * 就诊病区
	 * @param Name_dep_nur
	 */
	public void setName_dep_nur(String Name_dep_nur) {
		setAttrVal("Name_dep_nur", Name_dep_nur);
	}
	/**
	 * 住院号
	 * @return String
	 */
	public String getCode_amr_ip() {
		return ((String) getAttrVal("Code_amr_ip"));
	}
	/**
	 * 住院号
	 * @param Code_amr_ip
	 */
	public void setCode_amr_ip(String Code_amr_ip) {
		setAttrVal("Code_amr_ip", Code_amr_ip);
	}
}